traffic_ops/testing/api_contract/v4/pytest.ini

```diff
@@ -0,0 +1,6 @@
+[pytest]
```

Comment:
file missing license header

Reply:
Added license header in pytest.ini file.